addons/crm/tests/test_crm_lead_convert_mass.py

```diff
@@ -42,7 +42,7 @@ def test_assignment_salesmen_wteam(self):
         with self.assertQueryCount(user_sales_manager=0):
             test_leads = self.env['crm.lead'].browse(test_leads.ids)
 
-        with self.assertQueryCount(user_sales_manager=583):  # crm 543 / com 545 / ent 583
+        with self.assertQueryCount(user_sales_manager=585):  # crm 543 / com 545 / ent 583
```

Comment:
What are the extra queries exactly?
Is that one for each notified user?

Can't we reduce them, by preparing fetching all necessary data for all users beforehand?

Reply:
it comes from the `self.env['res.partner'].browse(partner_id)` but indeed, we got enough data beforehand and I will see if it can stay the same by doing it correctly.